vespalib/src/vespa/vespalib/net/tls/crypto_codec_adapter.cpp

```diff
@@ -163,4 +170,28 @@ CryptoCodecAdapter::flush()
     return 0; // done
 }
 
+ssize_t
+CryptoCodecAdapter::half_close()
+{
+    auto flush_res = flush_all();
+    if ((flush_res < 0)) {
```

Comment:
Extra hug from parentheses that might be unintended? (also below)

Reply:
yes, extra hugs left behind by removal of bad idea